pkg/juicefs/mount/process_mount.go

```diff
@@ -108,13 +108,15 @@ func (p *ProcessMount) JDeleteVolume(ctx context.Context, jfsSetting *jfsConfig.
 
 	var existed bool
 
-	if _, err := util.DoWithinTime(ctx, defaultCheckTimeout, nil, func() (err error) {
+	if err := util.DoWithTimeout(ctx, defaultCheckTimeout, func() (err error) {
 		existed, err = k8sMount.PathExists(volPath)
 		return err
 	}); err != nil {
 		return fmt.Errorf("could not check volume path %q exists: %v", volPath, err)
 	} else if existed {
-		stdoutStderr, err := p.RmrDir(ctx, volPath, jfsSetting.IsCe)
```

Comment:
This may take a long time to delete a volume with many files, remove this timeout.

Reply:
Removed